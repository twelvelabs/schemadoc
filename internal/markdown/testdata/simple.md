A _simple_ document with a bit of [markup](#markup).

Another paragraph with some `inline code`.


* one
* two
* three
