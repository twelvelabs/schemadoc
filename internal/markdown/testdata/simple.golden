<p>A <em>simple</em> document with a bit of <a href="#markup">markup</a>.<p>Another paragraph with some <code>inline code</code>.<ul><li>one<li>two<li>three</ul>